odl/test/tomo/geometry/geometry_test.py

```diff
@@ -576,17 +588,39 @@ def test_helical_cone_flat_props(shift):
     assert all_almost_equal(geom.det_axes(0), ([1, 0, 0], [0, 0, 1]))
     assert all_almost_equal(geom.translation, translation)
 
+    # Here we expect to rotate the reference point to [-det_rad, 0, 0] and
+    # then shift by (pi/4, pi/4) (=detector param) along the detector axes
+    # at that angle. In addition, everything is shifted along the rotation
+    # axis [0, 0, 1] by 1/4 of the pitch
+    # (since the pitch is the vertical distance after a full turn 2*pi).
+
     # Check that we first rotate, then shift along the initial detector
     # axes rotated according to the angle, which is equivalent to shifting
     # first and then rotating.
-    # Here we expect to rotate the reference point to [-det_rad, 0, 0] and
-    # then shift by (1, 1) (=detector param) along the detector axes
-    # ([0, 1, 0], [0, 0, 1]) at that angle. In addition, everything is
-    # shifted along the rotation axis [0, 0, 1] by 1/4 of the pitch
-    # (since the pitch is the vertical distance after a full turn 2*pi).
-    # Global translation should come last.
-    assert all_almost_equal(geom.det_point_position(np.pi / 2, [1, 1]),
-                            translation + [-det_rad, 1, 1 + pitch / 4])
+    # Here we expect to rotate the reference point to [-det_rad, 0, 0] and,
+    # in the case of flat detector, shift by (1, 1) (=detector param)
+    # along the detector axes ([0, 1, 0], [0, 0, 1]) at that angle.
+    # In the case of curved detectors, curvature is taken into account.
+    # In addition, everything is shifted along the rotation axis [0, 0, 1]
+    # by 1/4 of the pitch (since the pitch is the vertical distance after
+    # a full turn 2*pi). Global translation should come last.
```

Comment:
There is only this one sentence about curved detectors, that "curvature is taken into account". Other things apply to all detectors.

I don't know if it makes sense to explain in words, how curvature is taken into account, since the formulas should speak for themselves...

Reply:
I guess it should be enough to state that "curvature is included in the model for the detector geometry".

Reply:
Since this is just a comment to test case, I think it can stay as is..